content/en/blog/minikube-falco-kernel-module.md

```diff
@@ -0,0 +1,142 @@
+using Minikube >= 1.8.0 (with the minikube iso, e.g: using a VM driver) will also find installed a copy of the Falco kernel module ready to serve Falco instances in user-space!
+
+At the moment of writing, Minikube installs Kubernetes 1.17.3, and the Falco repository does not yet ship the manifests containing the new changes for
+Kubernetes >= 1.16, altough there are PRs [#1044](https://github.com/falcosecurity/falco/pull/1044) and [#1005](https://github.com/falcosecurity/falco/pull/1005) that go to that direction.
+
+For this reason, to test this you can either just patch the manifests yourself or use the Falco Helm Chart that was [already patched](https://github.com/helm/charts/pull/17339) and works in this case!
+
+It is extremely easy! After [installing Minikube](https://minikube.sigs.k8s.io/docs/start/) 1.8.0 you only need to start a cluster and [install Falco using the Helm Chart](https://github.com/helm/charts/tree/master/stable/falco).
+
+
+Create the cluster with Minikube using a VM driver, in this case Virtualbox:
+
+```bash
+minikube start --driver=virtualbox
+```
+
+Once you issue the command, look at the logs to make sure you have at least Minikube 1.8.0:
+
+```
+😄  minikube v1.8.1 on Arch 
+    ▪ KUBECONFIG=/home/fntlnz/.kube/current
+✨  Automatically selected the virtualbox driver
+💿  Downloading VM boot image ...
+🔥  Creating virtualbox VM (CPUs=2, Memory=3939MB, Disk=20000MB) ...
+🐳  Preparing Kubernetes v1.17.3 on Docker 19.03.6 ...
+🚀  Launching Kubernetes ... 
+🌟  Enabling addons: default-storageclass, storage-provisioner
+⌛  Waiting for cluster to come online ...
+🏄  Done! kubectl is now configured to use "minikube"
+```
+
+Now that our VM is ready, we can SSH into it to verify that the module is there!:
+
+
+```
+minikube ssh
+```
+
+Once we are in:
+
+```
+lsmod | grep -i falco
+```
+
+Aaaand..... It will show nothing! Because the Falco module has not been loaded yet!
+
+Let's continue with the installation of Falco, it will load the module for us afterwards.
+
+Verify that everything is running, it will need to show the system pods:
+
+```bash
+kubectl get pods --all-namespaces
+```
+
+Add the Stable chart repository to Helm:
+
+```bash
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+```
+
+Install Falco using Helm:
+
+```bash
+helm install falco stable/falco
+```
+
+It will give something like this:
+
+```
+NAME: falco
+LAST DEPLOYED: Sun Mar  8 16:29:11 2020
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+Falco agents are spinning up on each node in your cluster. After a few
+seconds, they are going to start monitoring your containers looking for
+security issues.
+
+No further action should be required.
+```
+
+Read the logs to make sure Falco is running!
+
+
+```bash
+kubectl logs -l app=falco -f
+```
+
+It will show errors trying to find the Kernel headers, those are needed to build the module but we can just skip those errors because the module is already there!
```

Comment:
Can we update the Falco messaging to reflect this? I think we should be able to detect if we are running in Minikube and load accordingly.